blog/models.py

```diff
@@ -1,3 +1,23 @@
 """Models for project."""
 
-from django.db import models  # noqa: F401
+from django.db import models
+from django.utils import timezone
+
+
+class Post(models.Model):
+    """Model that represents post in our blog."""
+
+    author = models.ForeignKey('auth.User', on_delete=models.CASCADE)
+    title = models.CharField(max_length=200)
+    text = models.TextField()
+    created_date = models.DateTimeField(default=timezone.now)
+    published_date = models.DateTimeField(blank=True, null=True)
+
+    def publish(self):
+        """Publish the post."""
+        self.published_date = timezone.now()
+        self.save()
+
+    def __str__(self):
+        """Return title of Post."""
```

Comment:
I see that you do the return in the next line. The docstring should point to the effect of calling the method, rather than the implementation details. Better docstring would sound like `Render the post instance as a string.`

Reply:
@webknjaz done :)